scripts/updates/utils/termux_pkg_upgrade_version.sh

```diff
@@ -122,32 +124,41 @@ termux_pkg_upgrade_version() {
 		fi
 	done < "${TERMUX_SCRIPTDIR}/scripts/big-pkgs.list"
 
-	_termux_should_cleanup "${big_package}" && "${TERMUX_SCRIPTDIR}/scripts/run-docker.sh" ./clean.sh
+	if [[ -z "${TERMUX_PKG_AUTO_UPDATE_GROUP}" ]]; then
+		_termux_should_cleanup "${big_package}" && "${TERMUX_SCRIPTDIR}/scripts/run-docker.sh" ./clean.sh
+
+		if ! "${TERMUX_SCRIPTDIR}/scripts/run-docker.sh" ./build-package.sh -C -a "${TERMUX_ARCH}" -i "${TERMUX_PKG_NAME}"; then
+			_termux_should_cleanup "${big_package}" && "${TERMUX_SCRIPTDIR}/scripts/run-docker.sh" ./clean.sh
+			git checkout -- "${TERMUX_PKG_BUILDER_DIR}"
+			termux_error_exit "ERROR: failed to build."
+		fi
 
-	if ! "${TERMUX_SCRIPTDIR}/scripts/run-docker.sh" ./build-package.sh -C -a "${TERMUX_ARCH}" -i "${TERMUX_PKG_NAME}"; then
 		_termux_should_cleanup "${big_package}" && "${TERMUX_SCRIPTDIR}/scripts/run-docker.sh" ./clean.sh
-		git checkout -- "${TERMUX_PKG_BUILDER_DIR}"
-		termux_error_exit "ERROR: failed to build."
 	fi
 
-	_termux_should_cleanup "${big_package}" && "${TERMUX_SCRIPTDIR}/scripts/run-docker.sh" ./clean.sh
-
 	if [[ "${GIT_COMMIT_PACKAGES}" == "true" ]]; then
-		echo "INFO: Committing package."
+		echo "INFO: Committing package${TERMUX_PKG_AUTO_UPDATE_GROUP:+" to ${TERMUX_PKG_AUTO_UPDATE_GROUP} update group"}."
 		stderr="$(
+			if [[ -n "${TERMUX_PKG_AUTO_UPDATE_GROUP}" ]]; then
+				# Switch to auto-update branch, create if needed
+				git checkout -b "auto-update/${TERMUX_PKG_AUTO_UPDATE_GROUP}/${GITHUB_RUN_ID}" || \
+					git checkout "auto-update/${TERMUX_PKG_AUTO_UPDATE_GROUP}/${GITHUB_RUN_ID}"
```

Comment:
```suggestion
				git checkout "auto-update/${TERMUX_PKG_AUTO_UPDATE_GROUP}/${GITHUB_RUN_ID}"
```

Reply:
Indentation is needed to make it explicit it is a part of another command (it goes after `\` in the previous line).